tests/e2e-upgrade/upgrade/chainsaw-step-60-assert-1-1.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Why is this assert (60-assert.yaml) split into 3 different files?

Reply:
Fixed